Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/TransferLogSink.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Frends.SFTP.UploadFiles.Definitions
+{
+    /// <summary>
+    /// Sink that is used to store messages and events from seriolog Logger
+    /// </summary>
+    public class TransferLogSink : ILogEventSink
```

Comment:
If TransferLogSink is set to internal, it cannot implement the ILogEventSink interface which is public.

Reply:
I do not think that this is true, you can definitely implement public interfaces by internal classes. What error are you getting?

Reply:
I get error code:
`CS0737	'TransferLogSink' does not implement interface member 'ILogEventSink.Emit(LogEvent)'. 'TransferLogSink.Emit(LogEvent)' cannot implement an interface member because it is not public.`